trade-finance-manager-api/src/v1/controllers/deal.party-db.ff-test.js

```diff
@@ -130,6 +124,19 @@ it('should allow a probability of default set to its default value', async () =>
   expect(result).toBe('TEST_URN');
 });
 
+it('should allow a probability of default set to undefined', async () => {
+  getOrCreatePartyDbInfo.mockResolvedValue([{ partyUrn: 'TEST_URN' }]);
+
+  const companyData = { companyRegNo: '12345678', companyName: 'name', probabilityOfDefault: undefined };
```

Comment:
Can an assertion please be added without `probabilityOfDefault` field in the body.